landfill/functional_tests/debug/t3_sentry_userproc_win.yea

```diff
@@ -1,3 +1,4 @@
+# This file has not been migrated to test_sentry.py because there is no specific windows unit tests.
```

Comment:
we actually run all the unit tests on windows in main see here: https://github.com/wandb/wandb/blob/main/.circleci/config.yml#L497-L508

Reply:
Hmm interesting. I dont see it actually running in the checks here https://app.circleci.com/pipelines/github/wandb/wandb/39079/workflows/172218e8-b348-4043-9c94-2811e8c223b8

am I missing something? Or it is possibly misconfigured?

Reply:
oh wait it looks like it only runs on certain branches. Got it!
In that case we can also remove the other test.

Reply:
if you want to test it before you merge you can temporarily mock with the config file